pkg/util/testingjobs/job/wrappers.go

```diff
@@ -134,12 +134,8 @@ func (j *JobWrapper) Request(r corev1.ResourceName, v string) *JobWrapper {
 }
 
 func (j *JobWrapper) Image(name string, image string, args []string) *JobWrapper {
-	j.Spec.Template.Spec.Containers[0] = corev1.Container{
-		Name:      name,
-		Image:     image,
-		Args:      args,
-		Resources: corev1.ResourceRequirements{Requests: corev1.ResourceList{}},
-	}
+	j.Spec.Template.Spec.Containers[0].Image = image
+	j.Spec.Template.Spec.Containers[0].Args = args
```

Comment:
This was resetting the requests :(

Reply:
.. we should also o something about the name 

1. remove it , since it is not used , 
2. use it to
  a.  change the fist container's name also (preserve behavior)
  b.  locate the container (probably overkill)

2a is the easiest, 1 probably the best

Reply:
ah, yes, I'll just remove it

Reply:
wait, I'll do that in a follow up, to reduce the diff.

Reply:
Applied 2a